operators/pkg/forge/containers.go

```diff
@@ -467,6 +458,20 @@ func ContainerVolume(volumeName, claimName string, environment *clv1alpha2.Envir
 	}
 }
 
+// TenantPersonalNFSVolume returns the Personal volume of the tenant
+func TenantPersonalNFSVolume(volumeName, nfsServerName, nfsPath string) corev1.Volume {
```

Comment:
```suggestion
func MyDriveVolume(volumeName, nfsServerName, nfsPath string) corev1.Volume {
```

A bit more consistent IMHO.

Reply:
Changed